tests/migration_test.go

```diff
@@ -856,7 +848,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 			var pvName string
 			var vmi *v1.VirtualMachineInstance
 			BeforeEach(func() {
-				tests.SkipMigrationTestIfRunnigOnKindInfra()
+				tests.SkipPVCTestIfRunnigOnKindInfra()
```

Comment:
AFAIU, this test is not solved by https://github.com/kubevirt/kubevirt/pull/3171. If so, please introduce a new skip message.

Reply:
thats right , but maybe 3171 is mandatory as well or all that will be needed once its done,
(now it stuck on VM not getting scheduled), so what you suggest please to say in the skip message ?

Reply:
I suggest to create a new kubevirt issue telling the test fails and explaing what you know about it. And add a link to this issue in the skip. Similar to https://github.com/kubevirt/kubevirt/blob/master/tests/probes_test.go#L51